packages/runtime/container-runtime/src/rootGcHandle.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IFluidHandle, IFluidHandleContext } from "@fluidframework/core-interfaces";
+import { generateHandleContextPath } from "@fluidframework/runtime-utils";
+
+/**
+ * Handle for root objects in the graph. These are used only for representing root nodes in the garbage collection
+ * reference graph.
+ */
+export class RootGCHandle implements IFluidHandle {
```

Comment:
It looks very similar to FluidObjectHandle, with a difference that it's attached already.
Can we reuse code? Or make this class more generic and not specific to "root" part?

Also, it would be nice to have all handle implementations in one place (I assume E:\Git\FluidFramework3\packages\runtime\runtime-utils\src\remoteObjectHandle.ts is a better place?)